client/src/main/kotlin/io/spine/examples/chatspn/desktop/chat/ChatsPage.kt

```diff
@@ -0,0 +1,530 @@
+                    strokeWidth = 1.dp.toPx()
+                )
+            },
+    ) {
+        Row(
+            Modifier.padding(7.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            UserAvatar()
+            Text(
+                chat.get().name,
+                modifier = Modifier.padding(start = 5.dp),
+                style = MaterialTheme.typography.headlineLarge,
+            )
+        }
+    }
+}
+
+/**
+ * Represents the list of messages in the chat.
+ */
+@Composable
+private fun ChatMessages(
+    model: ChatsPageModel,
+    selectedChat: ChatId
+) {
+    val messages by model
+        .messages(selectedChat)
+        .collectAsState()
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(5.dp, 0.dp),
+        verticalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        messages.forEach { message ->
+            item(message.id) {
+                ChatMessage(model, message)
+            }
+        }
+        item {
+            Box(Modifier.height(50.dp))
+        }
+    }
+}
+
+/**
+ * Represents the single message view in the chat.
+ */
+@Composable
+private fun ChatMessage(
+    model: ChatsPageModel,
+    message: MessageData
+) {
+    val isMyMessage = message.sender.id
+        .equals(model.authorizedUser().id)
+    Box(
+        modifier = Modifier.fillMaxWidth(),
+        contentAlignment = if (isMyMessage) Alignment.CenterEnd else Alignment.CenterStart
+    ) {
+        Surface(
+            modifier = Modifier.padding(4.dp),
+            shape = RoundedCornerShape(size = 20.dp),
+            elevation = 8.dp,
+            color = MaterialTheme.colorScheme.surface
+        ) {
+            MessageContent(message)
+        }
+    }
+}
+
+/**
+ * Represents the content the message.
+ */
+@Composable
+private fun MessageContent(message: MessageData) {
+    Row(Modifier.padding(10.dp), verticalAlignment = Alignment.Top) {
+        UserAvatar()
+        Spacer(Modifier.size(8.dp))
+        Column {
+            Row {
+                SenderName(message.sender.name)
+                Spacer(Modifier.size(10.dp))
+                PostedTime(message.whenPosted)
+            }
+            Spacer(Modifier.size(8.dp))
+            MessageText(message.content)
+        }
+    }
+}
+
+/**
+ * Represents the name of the user who sent this message.
+ */
+@Composable
+private fun SenderName(username: String) {
+    Text(
+        text = username,
+        style = MaterialTheme.typography.headlineMedium
+    )
+}
+
+/**
+ * Represents the time when the message was posted.
+ */
+@Composable
+private fun PostedTime(time: Timestamp) {
+    Text(
+        text = time.toStringTime(),
+        style = MaterialTheme.typography.headlineSmall,
+        color = MaterialTheme.colorScheme.onSecondary
+    )
+}
+
+/**
+ * Represents the text of the message.
+ */
+@Composable
+private fun MessageText(text: String) {
```

Comment:
I think that this component is redundant. Consider moving it to the `MessageContent`.